omelet-core/src/main/java/omelet/driver/DriverInitialization.java

```diff
@@ -90,106 +75,163 @@ public void afterInvocation(IInvokedMethod method, ITestResult testResult) {
 
 			if (beforeMethodPresent) {
 				if (!afterMethodShouldbePresent) {
-					cleanup(method, testResult);
+					cleanupTest(method, testResult);
 				}
 			} else {
-				cleanup(method, testResult);
-			}
-		}
-		// Check for AfterMethod if present check for browser and quit
-		if (method.getTestMethod().isAfterMethodConfiguration()) {
-			if (Driver.driver.get() != null) {
-				cleanup(method, testResult);
+				cleanupTest(method, testResult);
 			}
 		}
+//        // Check for AfterMethod if present check for browser and quit
+//        if (method.getTestMethod().isAfterMethodConfiguration()) {
+//            if (DriverManager.driver.get() != null) {
+//                cleanup(method, testResult);
+//            }
+//        }
 	}
```

Comment:
Driver.tearDown is removed from AfterMethod , then in the existing parallel functionality how the new browser will be initialized ?

Right now i can see Driver clean up is only after suite